.github/workflows/rtr_syscheck.yml

```diff
@@ -9,7 +9,6 @@ on:
       - "src/unit_tests/syscheckd/**"
       - "src/Makefile"
       - ".github/workflows/rtr_syscheck.yml"
-      - ".github/actions/**"
```

Comment:
We should remove this changes, or at least improve the filtering of these workflows.